### PR TITLE
view questions asked

### PR DESCRIPTION
**Pull request by tracyee0**
created a new component liked to the nav where user can view asked questions and see the answers

### CODE REVIEW

File: src/app/background/background.component.ts [hunk 1]
```diff
@@ -10,7 +10,9 @@ export class BackgroundComponent implements OnInit {
   public questionObject =
```
Comment (texyh) [inline src/app/background/background.component.ts]: can you delete this? we are not using this anymore

File: src/app/view-questions/question.component.1.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+  constructor() { }
+
+  @Input()
+  public question: any; // {subject: 'example', author: 'emeka'}
+
+  ngOnInit() {
+  }
+
+  age() {
```
Comment (texyh) [inline src/app/view-questions/question.component.1.ts]: please delete this

**Review by texyh — COMMENTED** [review]
i can see question.component.1.ts and question.component.2.ts and so on in the view question component, what are they doing there? what is it used for? whats are they all the same files with different names?

File: src/app/view-questions/view-questions.component.css [hunk 3]
```diff
@@ -0,0 +1,3 @@
+h2{
+    text-align: center;
```
Comment (texyh) [inline src/app/view-questions/view-questions.component.css]: bootstrap has a class that can do this for you text-center

File: src/app/view-questions/view-questions.component.html [hunk 4]
```diff
@@ -0,0 +1,28 @@
+<app-header></app-header>
+
+<div class="container">
+  <h2>Asked Questions</h2>
+  <div class="list-group">
+    <a href="#" class="list-group-item">
```
Comment (texyh) [inline src/app/view-questions/view-questions.component.html]: what are this for? are they the answers for the asked question? if so make them a component and loop over them as we did for question

**Comment by tracyee0** [comment]
yes the answer

**Pull request merged**